composer.json

```diff
@@ -21,7 +21,7 @@
     },
     "conflict": {
         "phpstan/phpdoc-parser": "<=0.5.3",
-        "phpstan/phpstan": "<=0.12.82",
+        "phpstan/phpstan": "<=0.12.88",
```

Comment:
This seems a bit confusing.
How can composer require 0.12.86 and conflict with 0.12.88 and lower at the same time?

Reply:
my bad, I looked in the wrong section